frontend/pages/ManageControlsPage/SetupExperience/cards/SetupAssistant/components/AdvancedOptionsForm/_styles.scss

```diff
@@ -0,0 +1,18 @@
+.advanced-options-form {
+  display: flex;
+  flex-direction: column;
+  gap: $pad-large;
```

Comment:
could also just add `.form` to the classNames to get the global styles to cover these 3, not important